cmd/argocd/commands/applicationset.go

```diff
@@ -230,6 +230,7 @@ func NewApplicationSetListCommand(clientOpts *argocdclient.ClientOptions) *cobra
 func NewApplicationSetDeleteCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command {
 	var (
 		noPrompt bool
+		cascade  bool
```

Comment:
Not sure if i understand, but where is usage for this variable?

Reply:
> Not sure if i understand, but where is usage for this variable?

as a flag, we are using in command.Flags